api/src/pcapi/routes/public/individual_offers/v1/bookings.py

```diff
@@ -194,6 +194,50 @@ def validate_booking_by_token(token: str) -> None:
     bookings_api.mark_as_used(booking)
 
 
+@blueprint.v1_bookings_blueprint.route("/keep/token/<token>", methods=["PATCH"])
+@spectree_serialize(
+    on_success_status=204,
+    api=blueprint.v1_bookings_schema,
+    tags=[constants.BOOKING_TAG],
+    resp=SpectreeResponse(
+        **(
+            BOOKINGS_API_BASE_CODE_DESCRIPTIONS
+            | {
+                "HTTP_204": (None, "The booking's validation has been cancelled successfully"),
+                "HTTP_403": (
+                    None,
+                    "This booking's validation cannot cancelled because you do not have the proper right or because the token's status is not used",
```

Comment:
```suggestion
                    "This booking's validation cannot be cancelled because you do not have the proper rights or because the token's status is not used",
```